src/irr/asset/CImageLoaderOpenEXR.h

```diff
@@ -32,6 +46,64 @@ namespace irr
 			asset::SAssetBundle loadAsset(io::IReadFile* _file, const asset::IAssetLoader::SAssetLoadParams& _params, asset::IAssetLoader::IAssetLoaderOverride* _override = nullptr, uint32_t _hierarchyLevel = 0u) override;
 
 		private:
+
+			struct SContext
+			{
+				constexpr static std::array<char, 4> magicNumber = { 0x76, 0x2f, 0x31, 0x01 };
```

Comment:
watch out, magic numbers are usually stored as `uint32_t` for the purpose of endianness check, see my SPIRV loader